go/src/koding/db/mongodb/modelhelper/deletedmember.go

```diff
@@ -0,0 +1,78 @@
+package modelhelper
+
+import (
+	"koding/db/models"
```

Comment:
`goimports` usually fail to sort it right, it think it's caused that both `koding/db/models` and  `gopkg.in/mgo.v2` look like external depedency.

I think in 1.6 it may sort vendored imports correctly, but well I'm groupping them by hand, correcting after goimports when it fails to do it for me.